ferret/explainers/__init__.py

```diff
@@ -58,7 +68,7 @@ def compute_feature_importance(
     def __call__(
         self,
         text: str,
-        target: int,
+        target: int = 1,
```

Comment:
This is a strong assumption, I think we should get rid of it (regardless of whether it's in the `main` branch or not already). Also note that in the task-API branch, `target` can also be a string and that type info should be 
```python
target: Union[str,int],
```

Reply:
This change should be completely removed. It was a remainder of trying to figure out how to make test_explainers.py work without crashes. In the end, we used target_pos_idx for testing so this change is unnecessary.